### PR TITLE
Use SendableChooser for selecting autonomous.

### LINKED ISSUE #25: projects should use SendableChooser for selecting autonomous commands

### PR DESCRIPTION
**Pull request by sciencewhiz**
Fixes #25

### CODE REVIEW

**Comment by sciencewhiz** [comment]
@SamCarlberg removing the Autonomous Command parameters makes this a lot cleaner. What do you think?

**Comment by SamCarlberg** [comment]
I can't believe we weren't generating SendableChoosers before.

I'll take a look at this tomorrow

**Comment by sciencewhiz** [comment]
They were added to eclipse Command Based last year, but not Robot Builder. In our code, we'd use SendableChooser manually.

File: src/main/resources/export/java/RobotTemplate-autonomous.java [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #set($command = $robot.getProperty("Autonomous Command").getValue())
-#set($params = $robot.getProperty("Autonomous command parameters").getValue())
 
-#if( $command != "None" )        autonomousCommand = #command_instantiation( $command $params );
+#foreach( $component in $components )
+#if ($component.getBase().getType() == "Command"
+     && $component.getProperty("Autonomous Selection").getValue())
+        chooser.addObject("$component.getName()", new #class($component.getName())());
```
Comment (SamCarlberg) [inline src/main/resources/export/java/RobotTemplate-autonomous.java]: This doesn't support parameterized commands. We should add one autonomous choice per preset. Parameterized commands with no presets shouldn't be added

For example, if I have a command with parameters `x` and `y`, and various presets for those parameters, each preset should appear in the chooser.

**Comment by SamCarlberg** [comment]
Setting a parameterized command as the autonomous command also breaks. Should add a property to the robot for a parameter preset to use for the autonomous command

**Pull request merged**